R/git.R

```diff
@@ -85,6 +85,22 @@ git_fetch <- function(root = NULL) {
 }
 
 git_pull <- function(root = NULL) {
-  orderly_log("git", "fetch")
+  orderly_log("git", "pull")
   git_run("pull", root = root, check = TRUE)
 }
+
+git_branches_no_merged <- function(root = NULL, include_master = FALSE) {
+  branches <- git_run(c("for-each-ref", "--sort=-committerdate",
+                        "--format='%(refname:short),%(committerdate)'",
+                        "--no-merged=master"), root = root, check = TRUE)$output
```

Comment:
How do we deal with old stale branches? There must be a bunch of dead ends in montagu-reports that we'd not really want to show? But it's not the end of the world either?

Reply:
Stale branches would be shown at the moment. In some post processing we could just exclude branches which `last_commit` is older than a certain time. What would be appropriate? A couple of months?

Reply:
Let's go with everything for now, and later we can process that via the runner